src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
                 .description("Number of exams starting within the next minutes multiplied with students in the course").register(meterRegistry);
     }
 
```

Comment:
Here you could add some javadoc
```suggestion
    /**
     * Register metrics for active admin accounts
     **/
```